tests/kafkatest/services/streams.py

```diff
@@ -43,6 +45,132 @@ class StreamsSmokeTestBaseService(KafkaPathResolverMixin, Service):
         "streams_stderr": {
             "path": STDERR_FILE,
             "collect_default": True},
+        "streams_log.0": {
+            "path": LOG_FILE + ".0",
+            "collect_default": True},
+        "streams_stdout.0": {
+            "path": STDOUT_FILE + ".0",
+            "collect_default": True},
+        "streams_stderr.0": {
+            "path": STDERR_FILE + ".0",
+            "collect_default": True},
+        "streams_log.1": {
+            "path": LOG_FILE + ".1",
+            "collect_default": True},
+        "streams_stdout.1": {
+            "path": STDOUT_FILE + ".1",
+            "collect_default": True},
+        "streams_stderr.1": {
+            "path": STDERR_FILE + ".1",
+            "collect_default": True},
+        "streams_log.0-1": {
```

Comment:
Yes. But we don't know which one we get because we shuffle the order or processors for rebalancing. The once that don't exists are just not collected, but we need to list them all :(

I am more than happy to change the code, but I don't know a better solution... Is there anything better we could do (I still don't know all ducktape magic)

Reply:
Although we shuffle the processes, each of them will still be bounced exactly twice, once per round right? In this case could we just use `streams_log.0` and `streams_log.1` on each processor instance?

Reply:
We could. I choose the current naming schema, because it encodes the rebalance order in the file names, what makes debugging easier. Otherwise, you have to extract this information from the ducktape log. Let me know if you think this simplification is worth it, or if you prefer removing the counter reducing the log files to 3.

Reply:
I see your point now, that makes sense. Let's keep it as is.